service-commands/scripts/provision-dev-env.sh

```diff
@@ -113,12 +119,12 @@ function setup_profile() {
     echo 'export AUDIUS_REMOTE_DEV_HOST=$(curl -sfL -H "Metadata-Flavor: Google" http://metadata/computeMetadata/v1/instance/network-interfaces/0/access-configs/0/external-ip)' >> $HOME/.profile
     echo 'export AAO_DIR=$HOME/anti-abuse-oracle' >> $HOME/.profile
     echo 'export TN_DIR=$HOME/trusted-notifier-service' >> $HOME/.profile
-    echo '
-if [ -d ~/.aliases ]; then
-  for f in ~/.aliases/*; do
-    source $f
-  done
-fi' >> $HOME/.profile
+        
+    if [ -d $PROTOCOL_DIR/service-commands/scripts/.env/.aliases ]; then
+        for f in $PROTOCOL_DIR/service-commands/scripts/.env/.aliases/*; do
+            tail -n +2 $f >> $HOME/.bash_aliases
```

Comment:
What we're trying to minimize is the customizations of all environmental tooling and instead have people opt-into customized setups.

Instead, if we place the requested changes into [.env/.aliases/10.audius_aliases](https://github.com/AudiusProject/audius-protocol/blob/master/service-commands/scripts/.env/.aliases/10.audius_aliases), people can pull the latest updates into their own environment instead of aliases being swapped out silently.

Reply:
@isaacsolo after looking at a newly launched setup, let's move these new aliases into [.env/.aliases/10.audius_aliases](https://github.com/AudiusProject/audius-protocol/blob/master/service-commands/scripts/.env/.aliases/10.audius_aliases).

In order to load these aliases, we need to make sure the ~/.aliases folder exists/has been copied and that this has been run to add these lines into ~/.profile:

```bash
    echo '
if [ -d ~/.aliases ]; then
  for f in ~/.aliases/*; do
    source $f
  done
fi' >> $HOME/.profile
```

After a bit more investigating, zsh doesn't use ~/.profile and instead uses ~/.zprofile. Let me think more about how to make this a bit more seamless, but for now the proper location for these new aliases should be within 10.audius_aliases which should work after ~/.profile has been modified.

If it still doesn't work for you, we can screenshare again since sometimes the addition of some dotfiles can cause other dotfiles to no longer be read.

Reply:
i agree we should minimize customization. i think we should just get this merged in so we can have this working for bash at least. then you can get it working for both but i don't think ~/.profile is the way. we should do something like this https://medium.com/@uneetpatel2149/set-aliases-for-zsh-terminal-and-bash-in-linux-7a500f36ad1c.